### PR TITLE
FEAT: Create React App For The Project

### PR DESCRIPTION
**Pull request by ZHIHANCHEN03**
created react app for the proj and add install.md to guide installation

### CODE REVIEW

File: easy-contact/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (Rexezuge) [inline easy-contact/.gitignore]: `.gitignore` should be placed in the root directory

File: install.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Dependenices
```
Comment (Rexezuge) [inline install.md]: `install.md` should be named `INSTALL.md`

Comment (ZHIHANCHEN03) [reply install.md]: have updated the name

File: easy-contact/.gitignore (hunk 1 above)
Comment (ZHIHANCHEN03) [reply easy-contact/.gitignore]: moved the files to the root directory

**Review by xiaoshaoyc — COMMENTED** [review]
consider a better naming for the root folder. `easy-contact` is too general, and we have the backend in the same repo...

**Comment by ZHIHANCHEN03** [comment]
> consider a better naming for the root folder. `easy-contact` is too general, and we have the backend in the same repo...

Have updated name

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Dependenices
```
Comment (qit2) [inline INSTALL.md:1]: Can move the content in this file to README.md

Comment (ZHIHANCHEN03) [reply INSTALL.md:1]: Don't have to move the content for now

**Review by XiaoyuYang18 — APPROVED** [review]
Good Good

**Comment by wangz44** [comment]
make sense

**Comment by wangz44** [comment]
good commit

**Review by xiaokeai55 — APPROVED** [review]
Good

**Review by wangz44 — APPROVED** [review]
make sense

**Review by xiaoshaoyc — APPROVED** [review]

**Pull request merged**